lightfm/lightfm.py

```
@@ -16,6 +16,219 @@
+    def __mul__(self, other):
+        """Basic matrix multiplication for simple cases"""
+        # This is a simplified version - in real scipy this is much more complex
+        if hasattr(other, 'shape'):
+            # Matrix multiplication - return a simple result object
+            return SimpleMatrixResult(self, other)
+        else:
+            # Scalar multiplication
+            result = CsrMatrix(self.shape, self.dtype)
+            for key, value in self.data.items():
+                result.data[key] = value * other
+            return result
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)